File: app/src/main/java/com/example/movies/data/entity/SearchMovie.java
package com.example.movies.data.entity;

import java.util.ArrayList;

public class SearchMovie {
    private Number popularty;
    private int id;
    private boolean video;
    private int voteCount;
    private int voteAverage;
    private String title;
    private String releaseDate;
    private String originalLanguage;
    private String originalTitle;
    private ArrayList<Integer> genreids;
    private String backdropPath;
    private boolean adult;
    private String overview;
    private String poster_path;

    public Number getPopularty() {
        return popularty;
    }

    public int getId() {
        return id;
    }

    public boolean isVideo() {
        return video;
    }

    public int getVoteCount() {
        return voteCount;
    }

    public int getVoteAverage() {
        return voteAverage;
    }

    public String getTitle() {
        return title;
    }

    public String getReleaseDate() {
        return releaseDate;
    }

    public String getOriginalLanguage() {
        return originalLanguage;
    }

    public String getOriginalTitle() {
        return originalTitle;
    }

    public ArrayList<Integer> getGenreids() {
        return genreids;
    }

    public String getBackdropPath() {
        return backdropPath;
    }

    public boolean isAdult() {
        return adult;
    }

    public String getOverview() {
        return overview;
    }

    public String getPoster_path() {
        return poster_path;
    }
}

File: app/src/main/java/com/example/movies/data/entity/SearchMovie.kt
package com.example.movies.data.entity

import java.util.ArrayList

data class SearchMovie(val popularty: Number? = null,
                       val id: Int = 0,
                       val isVideo: Boolean = false,
                       val voteCount: Int = 0,
                       val voteAverage: Int = 0,
                       val title: String? = null,
                       val releaseDate: String? = null,
                       val originalLanguage: String? = null,
                       val originalTitle: String? = null,
                       val genreids: ArrayList<Int>? = null,
                       val backdropPath: String? = null,
                       val isAdult: Boolean = false,
                       val overview: String? = null,
                       val poster_path: String? = null)
